Sources/Sentry/Profiling/SpectoTime.mm

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Specto Inc. All rights reserved.
+
+#include "SpectoTime.h"
+
+#include <cassert>
+
+#ifdef __ANDROID__
```

Comment:
`m`: I'm pretty sure we won't run this code on Android. We can keep it, of course, if you want.